.github/workflows/run_tests.yml

```diff
@@ -2,7 +2,7 @@ name: Launch ACAS and run python tests
 
 on:
   push:
-    branches: ["main"]
+    branches: ["main", "release/*"]
```

Comment:
I just found a nice feature which is to use: $default-branch macro:
Example:
https://github.com/actions/starter-workflows/blob/main/ci/docker-image.yml#L5

https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/

Reply:
Ignore this comment.  I guess the $default-branch macro is for action templates, not for the actual GitHub actions yaml:

https://github.com/github/docs/discussions/5071